app/models/concerns/fog_extensions/libvirt/server.rb

```diff
@@ -33,6 +33,18 @@ def vm_description
         _("%{cpus} CPUs and %{memory} memory") % {:cpus => cpus, :memory => number_to_human_size(memory.to_i)}
       end
 
+      # Other Fog CRs use .interfaces as the accessor, but libvirt does not
```

Comment:
If you guys don't mind I'll try to submit these in Fog at some point then, `select_nic` is usually `.interfaces.get(id)` when you know the id, maybe it can be extended to handle network attributes too..